Examples/Io/Root/src/RootMeasurementWriter.cpp

```diff
@@ -141,9 +141,16 @@ ActsExamples::ProcessCode ActsExamples::RootMeasurementWriter::writeT(
           // Find the contributing simulated hits
           auto indices = makeRange(hitSimHitsMap.equal_range(hitIdx));
           // Use average truth in the case of multiple contributing sim hits
-          auto [local, pos4, dir] = averageSimHits(ctx.geoContext, surface,
-                                                   simHits, indices, logger());
-          dTree->fillTruthParameters(local, pos4, dir);
+          auto [local, pos4, dir, mom4] = averageSimHits(
+              ctx.geoContext, surface, simHits, indices, logger());
+          Acts::RotationMatrix3 rot =
+              surface
+                  .referenceFrame(ctx.geoContext, pos4.segment<3>(Acts::ePos0),
+                                  mom4.segment<3>(Acts::eMom0))
```

Comment:
I think direction would be sufficient here no?

Reply:
The only interface I see for Surface::referenceFrame is 
```
  virtual Acts::RotationMatrix3 referenceFrame(const GeometryContext& gctx,
                                               const Vector3& position,
                                               const Vector3& momentum) const;
```
Or am I misunderstanding?

Reply:
Hm I thought I renamed all these params to `direction` at some point... I believe the magnitude of that vector is not significant in general. Since we do not know the B field at this point I do not really see a reason for passing a magnitude and a unit vector forces the caller to give an actual direction and not a 0 vector

Reply:
oh yeah on main this should be renamed already

Reply:
Ah I see yeah. Changing now.

Reply:
Done